pkg/util/log/log.go

```diff
@@ -7,7 +7,7 @@
 // supports logging to multiple destinations, buffering messages logged before
 // setup, and scrubbing secrets from log messages.
 //
-// Compatibility
+// # Compatibility
```

Comment:
?

Reply:
I saw this and it changed it back, I think it might've been my autoformatter or something, I'll change it back.

Reply:
Update: this is added by gofmt